nix/all-engines.nix

```diff
@@ -53,13 +54,15 @@ in
       cp target/release/prisma-fmt $out/bin/
       cp target/release/libquery_engine${libSuffix} $out/lib/libquery_engine.node
     '';
+
+    dontStrip = true;
```

Comment:
There's some bug or something, without this building would currently fail with:
```
/nix/store/sgdq3c3lp6qabfwnrq26qzvrxx101hgi-strip.sh: line 80: exit_code: unbound variable
```
Interestingly it only seems to happen for these derivations for me, not when building other stuff, so something to isolate and report where appropriate when we have time.

Luckily disabling stripping actually makes a lot of sense here anyway even if that were fixed: we don't want it when `profile = "debug"`, and we don't strictly need it when `profile = "release"` as those binaries are already stripped by cargo since we configure it in `Cargo.toml` anyway.